### DISCUSSION TITLE
Is it possible to make entry in jobs collection immediately as the job gets created, incase of Recurring jobs ?

### DISCUSSION THREAD

**Discussion by kapilbablani** [jobrunr/jobrunr#610] [Q&A]
@rdehuyss I am looking for a configuration in jobRunr where I can control when to put the entries in jobs collections for an recurring job. At this point of time, I see the entry in jobs collection appears just 5 to 10 seconds prior to the scheduled time. But in case my service is down the job run is getting missed.
Is there a way to handle this, please help ?

**Comment by rdehuyss** [comment]
@kapilbablani - this will be part of JobRunr 6 Pro.
